src/main/java/su/litvak/chromecast/api/v2/Channel.java

```diff
@@ -184,8 +160,41 @@ public void run() {
                     try {
                         close();
                     } catch (IOException e) {
-                        LOG.warn("Error while closing channel: {}", ioex.getLocalizedMessage());
+                        LOG.warn("Error while closing channel: {}", ioex);
+                    }
+                } catch (Exception e) {
+                    LOG.warn("Unknown exception while reading", e);
```

Comment:
Please replace word `exception` with `error` for uniformity and readability of logs.